CONTRIBUTING.md

```diff
@@ -126,6 +126,16 @@ Tests are run by jest. They can be run by either running `npm run test` or `npm
 
 Files should be linted prior to creating a pull request. The linter will make sure that the code follows the desired codestyle and formats the files as needed. The linter can be run by `npm run lint` or `npm run compile`
 
+Note that linter removes trailing spaces and blank lines. In case if they are essential use the following trick to preserve them
+
+```js
+const str = dedent`
```

Comment:
This feels very hacky.

Reply:
I checked the source code of the plugin

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/20e959ffadeca007697534f1ad58c45d29cfdb04/rules/template-indent.js#L72

They do unconditional trimming.

I'll send them a PR to make it configurable

But honestly invisible trailing spaces are difficult to read anyway. With this "hack" we improve code readability

Reply:
I will take a closer look later on today.